tools/bakeoff/run_bakeoff.py

```
@@ -0,0 +1,160 @@
+    "depthwise_conv2d_3x3_bwd_weights": (2 * 32 * 64 * 56 * 56 * 4,
+                                         32 * 64 * 56 * 56 * 9 * 2),
+    "conv2d_1x1_bwd_weights":           ((16 * 64 * 28 * 28 + 16 * 64 * 28 * 28) * 4,
+                                         16 * 64 * 28 * 28 * 64 * 2),
+    "conv2d_3x3_bwd_weights":           ((8 * 32 * 28 * 28 + 8 * 64 * 28 * 28) * 4,
+                                         8 * 64 * 28 * 28 * 32 * 9 * 2),
+}
+
+
+def run_ours(dll):
+    """Parses --kernel-bench output into {kernel: (us, spread_pct)}."""
+    out = subprocess.run(["dotnet", dll, "--kernel-bench"],
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)